packages/react/src/contexts/TimerContext.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ScreenNames } from '@corbado/shared-ui';
```

Comment:
Do we really need a context for timers? The actual information when the next email can be sent comes from the backend (see BlockDataEmailVerify's "retryNotBefore" => this is the timestamp when the next retry is allowed). 

As this information will always be available to the EmailVerifyBlock (and also to the other blocks that needs some kind of resending logic, I don't think that we need additional state through a context.